studio-ui/static-assets/themes/cstudioTheme/css/styleicon.css

```
@@ -0,0 +1,203 @@
+	float: left;
+	padding: 10px 15px 0 15px;
+	width: 111px;
+}
+
+.iconName {
+	margin: 2px 0 0 0;
+	float: left;
+	min-width: 124px;
+}
+
+/*.iconNameR{
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)